server/remote_cache/cachetools/cachetools.go

```diff
@@ -27,6 +33,10 @@ const (
 	gRPCMaxSize        = int64(4000000)
 )
 
+var (
+	enableUploadCompresssion = flag.Bool("app.enable_upload_compression", false, "If true, enable compression of uploads to remote caches")
```

Comment:
The main place I was thinking of setting this was in the executor, and I was planning to add a flag alias for that.

Since it's a flag that controls client behavior, it feels wrong to me to put it in the cache section (which primarily controls serving behavior). Maybe i can just make it a var and define flags for it in the runner and in the executor? Open to ideas

Reply:
Having another section like `cache_client` might work, if that still feels awkward then that latter approach also sgtm (making it a var), though it might be slightly less error-prone to make it an argument to the BatchCASUploader rather than defining the var in this package (in case we inadvertently wind up trying to override the var via 2 separate flags at some point)

Reply:
I vote we add a `client` namespace under `cache` and put it there (`cache.client.enable_upload_compression`). I'm not clear on the need for aliases here, but certainly if they could clarify things for users I say go for it.